frontend/src/vault/deposit/DepositForm/chainOptionsConfig.ts

```diff
@@ -8,7 +8,7 @@ export type ChainWithAction = keyof typeof chainDepositOptionsConfig;
 
 export const chainDepositOptionsConfig = {
   thorchain: ['bond', 'unbond', 'leave', 'addPool', 'withdrawPool', 'custom'],
-  mayachain: ['bond', 'unbond', 'leave', 'custom'],
+  mayachain: ['bond_with_lp', 'unbond_with_lp', 'leave', 'custom'],
```

Comment:
We need to keep `bond` and `unbond` , because that's used by node operator as well